crates/snapbox/README.md

```diff
@@ -11,7 +11,7 @@
 - CLI stdout/stderr
 - Filesystem changes
 
-It is also flexible enough to build your own test harness like [trycmd](https://crates.io/crates/trycmd).
+It is also flexible enough to build your own test harness like [trycmd].
```

Comment:
I feel I'm missing something.  The syntax looks equivalent and the link seems to be working for me.

Reply:
Oh, sorry for lack of description 😅. It's the link on line 3 that was broken. I just unified the link with this one.

Reply:
Always interesting when diffs hide the important details